CHANGELOG-3.3.md

```diff
@@ -7,6 +7,9 @@ in 3.3 minor versions.
 To get the diff for a specific change, go to https://github.com/symfony/symfony/commit/XXX where XXX is the change hash
 To get the diff between two versions, go to https://github.com/symfony/symfony/compare/v3.3.0...v3.3.1
 
+* 3.3.7 (2017-08-18)
```

Comment:
new features are never added in patch versions. and this file is autogenerated, so you should not edit it.

Reply:
Thank you @stof. I've reverted the changes on this file.